src/components/map/Map.tsx

```diff
@@ -32,8 +32,8 @@ export const Map = (props: MapProps) => {
     () =>
       initialLocation && initialLocation?.resultType !== 'geolocation'
         ? initialLocation.coordinates
-        : currentLocation?.coordinates || FOCUS_ORIGIN,
-    [],
+        : getCurrentPosition()?.coordinates || FOCUS_ORIGIN,
+    [getCurrentPosition, initialLocation],
```

Comment:
Note: could be that the [useInitialCoordinates](https://github.com/AtB-AS/mittatb-app/blob/68ec54af07b9925613f2194e4c3cc56f11c9fe97/src/utils/use-initial-coordinates.ts) hook can simplify some things here when available